portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -33,14 +32,17 @@ public void store(Comment comment) {
      * @return An Iterable.
      */
     @Override
-    public List<Comment> load() {
+    public List<Comment> load(int limit) {
 
         Query query = new Query(KIND).addSort(TIMESTAMP, Query.SortDirection.DESCENDING);
-
         PreparedQuery results = this.datastoreService.prepare(query);
 
         List<Comment> comments = new ArrayList<>();
         for (Entity entity: results.asIterable()) {
+            if (comments.size() >= limit) {
```

Comment:
We won't need this if we can query the limit as well?

Reply:
Fixed

Reply:
Fixed